apps/omg_child_chain/lib/omg_child_chain/ethereum_event_aggregator.ex

```diff
@@ -74,7 +74,7 @@ defmodule OMG.ChildChain.EthereumEventAggregator do
     {:ok,
      %{
        # 200 blocks of events will be kept in memory
-       delete_events_threshold_height_blknum: 200,
+       delete_events_threshold_height_blknum: 1000,
```

Comment:
The name and value now strongly suggest to me it's block number... which is not. Or is it...

How about renaming to 'delete events threshold height', honestly I don't know what blknum has to do with events height... 😥

Comment says they are blocks of events, hmm

Reply:
uhh, touche, I need to take another look what it is, can't remember right now. and Ill update the docs